src/components/CheckEditor/FormComponents/RequestTargetInput.tsx

```diff
@@ -59,6 +52,7 @@ export const RequestTargetInput = ({
               placeholder={targetHelp?.example}
               data-fs-element="Target input"
               {...field}
+              value={typeof field.value === `string` ? field.value : ''}
```

Comment:
Yeah, this is one of those annoying ones where even though we know it can only be a string with the way we are using it we are adding verbosity to make the Typescript compiler stop complaining.

The problem lies with the loose type definition we ask for the `name` prop. Really we are only allowing very specific name fields here but it is a pain in the ass to write them so I'm going cheap and loose and paying for it this way. We can fix this at a later date if it is proving annoying 😃